info.json

```diff
@@ -0,0 +1,271 @@
+{
+  "name": "anomali-limo-feed",
+  "label": "Anomali ThreatStream TAXII Feed",
+  "description": "Anomali ThreatStream TAXII 2.0 Client",
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "version": "1.0.0",
+  "category": "SIEM",
+  "icon_small_name": "small_icon.png",
+  "icon_large_name": "large_icon.png",
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "title": "Server URL",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "server_url",
+        "value": "https://limo.anomali.com"
+      },
+      {
+        "title": "User Name",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "username",
+        "value": "guest"
+      },
+      {
+        "title": "Password",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "password",
+        "name": "password",
+        "value": "guest"
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_api_root_information",
+      "title": "Get API Root Information",
+      "description": "Returns information contains title, description, TAXII versions and maximum size of the content body.",
+      "category": "investigation",
+      "annotation": "get_api_root_information",
+      "output_schema": {
+        "max_content_length": "",
+        "title": "",
+        "versions": [
+          ""
+        ]
+      },
+      "parameters": [
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "get_collections",
+      "title": "Get Collections",
+      "description": "Returns a list of public/private/shared collection resources which users have the access.",
```

Comment:
Returns the list of public/private/shared collection resources to which the user has access.